File: 01《招聘一个靠谱的iOS》面试题参考答案/Demo_25题__objc_msgForward函数是做什么的_直接调用它将会发生什么/_objc_msgForward_demo/_objc_msgForward_demo/ViewController.m
//
//  ViewController.m
//  _objc_msgForward_demo
//
//  Created by http://weibo.com/luohanchenyilong/ (微博@iOS程序犭袁)on 15/9/10.
//  Copyright (c) 2015年 https://github.com/ChenYilong . All rights reserved.
//

#import "ViewController.h"
#import "Son.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [[Son alloc] init];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Swift/AnniversaryCalculator/AnniversaryCalculatorDemo/ViewController.swift
//
//  ViewController.swift
//  AnniversaryCalculatorDemo
//
//  Created by chenyilong on 2022/8/15.
//

import UIKit

class ViewController: UIViewController {
    let viewModel: ViewModel = .init()

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.

        for couple in viewModel.couples {
            if couple.shouldAnniversaryToRemind() {
                viewModel.couplesToRemind.append(couple)
            }
        }
        print(viewModel.couplesToRemind)
    }
}
